types/params_test.go

```diff
@@ -129,7 +130,25 @@ func TestConsensusParamsUpdate(t *testing.T) {
 			},
 			makeParams(100, 200, 300, 50, valSecp256k1),
 		},
+		{
+			makeParams(1, 2, 3, 0, valEd25519),
```

Comment:
I'm not familiar with this code, but this looks like a copy/paste bug

Reply:
This is testing the update functionality. It is fine as is.